docs/File-Formats.md

```diff
@@ -1263,20 +1263,20 @@ The cells contain the p-value for the GSVA score: A real number, between 0.0 and
+Treatment response data relate to outcome variables of treatments (compounds or combinations thereof) on samples (e.g., cell lines). cBioPortal supports any number of response data types imported from separate data files. Meta information on treatments (`stable_id`, `name`, `description` and `url`) are not part of the cBioPortal seed database, but are imported automatically from treatment response data files. For treatments already present in the database the `name`, `description` and `url` fields are overwitten by subsequent imports of treatment response data files. The corresponding `META:name`, `META:description` and `META:url` columns in treatment data files are optional. The `stable_id` and `META:name` fields must be unique within a data file. When the `META:name`, `META:description` and `META:url` fields are not defined in a data file, these fields will receive the value of the `stable_id` field. When multiple treatment data files are loaded as part of study, the `stable_id`, `META:name`, `META:description` and `META:url` columns must be identical is all data files.
 
 ### Treatment response meta file
-The meta file will be similar to meta files of other genetic profiles, such as mRNA expression. For treatment response data `RESPONSE-VALUE` is used as `datatype`. Any number of treatment response parameters can be loaded by means of different data files. For treatment meta files values for `stable_id` are not pre-registered in the cBioPortal implementation, but can be freely chosen by the user. Requirement is that each treatment meta file is assigned an unique `stable_id`.
+The meta file will be similar to meta files of other genetic profiles, such as mRNA expression. For treatment response data `LIMIT-VALUE` is used as `datatype`. The `LIMIT-VALUE` is validated to contain any continuous number optionally prefixed with a '>' or '<' threshold symbol. One or more treatment response data files can be loaded by means of different data files. For treatment meta files values for `stable_id` are not pre-registered in the cBioPortal implementation, but can be freely chosen by the user. Requirement is that each treatment meta file is assigned an unique `stable_id` by the user. This requirement is validated by the data validation scripts. 
```

Comment:
just realized that it is quite confusing to have `stable_id` in meta file and also as part of the "meta fields". These are two different stable_ids....maybe they should have different names? Maybe call the new one `entity_stable_id`?

Reply:
All occurences of `stable_id` were renamed to `entity_stable_id`.